kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/SessionManager.java

```diff
@@ -165,7 +163,8 @@ public void createEnvironment() {
                         resources.addAll(resourceInstaller.install(configUrl));
                     }
                 } else {
-                    log.warn("Did not find any kubernetes configuration.");
+                    log.warn(
+                        "Did not find any kubernetes configuration. If you are using fabric8-maven-plugin, ensure `mvn package fabric8:resource fabric8:build` is run first.");
```

Comment:
How about adding  `Did not find any kubernetes/openshift configuration.` instead of  `Did not find any kubernetes configuration.` to avoid confusing for end user.